portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,35 @@
 package com.google.sps.servlets;
 
+
 import java.io.IOException;
+import java.util.ArrayList;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
+  String[] listOfFacts = {"Fact #1: I'm a vegetarian.", 
+  "Fact #2: My hobbies include jailbreaking/modding, crochet, and playing video games.",
+  "Fact #3: I speak French (poorly)."
+};
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
```

Comment:
How about: 
```java
String[] facts = { ... };
Gson gson = new Gson();
String json = gson.toJson(facts);
return json;
```

It's good practice to keep the variable close to where it's being used. This helps when reading code (a person won't have to scroll or change their focus to find the value of the variable). Additionally, it helps with name conflicts (this isn't the case here). It's good to keep the cope of the variables close to each other.